scripts/imm_tests.bats

```diff
@@ -169,7 +169,7 @@ get_inference_accuracy(){
 
 @test "Delete tenant" {
     response="$(./imm rm t ${TENANT_NAME})"
-    grep -E "${TENANT_NAME} deleted" <<< $response
+    grep -E "DELETED" <<< $response
```

Comment:
I know that we currently in another tests only check if `TENANT_NAME` is on response, but maybe we could make it compatible to each other and also check in bats tests everywhere for `TENANT_NAME` and `status`?